content/docs/user-guide/troubleshooting.md

```diff
@@ -125,3 +125,53 @@ and retry the DVC command. Specifically, one of:
 
 [internal directories]:
   https://dvc.org/doc/user-guide/project-structure/internal-files
+
+## DVC Experiments may fail in Git shallow clones {#git-shallow}
+
+`dvc exp` commands use internal Git operations which may not work properly in
```

Comment:
maybe hyperlink to more info (presumably some section on reflog things?)

Reply:
Could use https://dvc.org/doc/user-guide/experiment-management/experiments-overview#how-does-dvc-track-experiments or https://dvc.org/blog/experiment-refs.

Reply:
We don't explain internal git merges anywhere, not even in Peter's blog post. Too deep